src/index.js

```diff
@@ -20,6 +20,7 @@ import Schema from './models/schema'
 import Selection from './models/selection'
 import State from './models/state'
 import Text from './models/text'
+import Range from './models/range'
```

Comment:
We are working on a new version of [`markup-it`](https://github.com/GitbookIO/markup-it/pull/32) which is using Slate for its data models, and we wanted to use `Range` when building `Text`.

Do you think we should not expose it?